src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -149,6 +150,25 @@ class FlushCommand : public ImageReplayerAdminSocketCommand {
 };
 
 template <typename I>
+class ResyncCommand : public ImageReplayerAdminSocketCommand {
```

Comment:
I just let it because you didn't confirm in the last comment. But I'm all in favor for removing it, and will do it.